l10n_ch_import_cresus/i18n/fr.po

```diff
@@ -0,0 +1,164 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+#	* account_import_cresus
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
```

Comment:
This seems to be the old translation files for 7.0. It might deserve a new export if you change any of the texts of this module